src/app/features/tech-record/components/tech-record-search-tyres/tech-record-search-tyres.component.ts

```diff
@@ -90,7 +91,10 @@ export class TechRecordSearchTyresComponent implements OnInit {
         this.form.controls['filter'].patchValue(v.filter);
         this.form.controls['term'].patchValue(v.term);
       });
-
+    this.referenceDataService.loadReferenceData(ReferenceDataResourceType.TyreLoadIndex);
+    this.referenceDataService
+      .getAll$(ReferenceDataResourceType.TyreLoadIndex)
+      .subscribe(data => (this.loadIndex = data as ReferenceDataTyreLoadIndex[]));
```

Comment:
Comment on this:
Either:
````ts
take(1)
````
Or
create a getter such as 
```ts
get loadIndex$() {
  return this.referenceDataService.getAll$(ReferenceDataResourceType.TyreLoadIndex)
```
and in the html this might work:
```html
<td class="govuk-table__cell">{{ tyre.axleLoadDouble | tyreAxleLoad : tyre.loadIndexTwinLoad : 4 : (loadIndex$ | async)| defaultNullOrEmpty }}</td>
```
I _think_ the second option would work and is preferable because you don't have to manage subscriptions that way.